inst/private/check_and_convert.m

```diff
@@ -0,0 +1,36 @@
+function obj = check_and_convert()
+  newl = sprintf('\n');
+  
+  str_check_is_sym = strjoin({'if temp is None or isinstance(temp, (sp.Basic, sp.MatrixBase)):',
+                              '  is_sym = True',
+                              'else:',
+                              '  is_sym = False'},newl);
+  pyexec(str_check_is_sym);
+  is_sym = pyeval('is_sym');
+  
+  if is_sym
+    ascii = pyeval('sp.pretty(_outs[0], use_unicode=False)');
+    unicode = pyeval('sp.pretty(_outs[0], use_unicode=True)');
+    str = pyeval('sympy.srepr(_outs[0])');
+    flat = pyeval('str(_outs[0])');
+    
+    str_eval = strjoin({'temp = Matrix([_outs[0]])',
```

Comment:
I'm not sure yet... perhaps this is related to `object_filter` in `python_header.py`?